exercises/aux_files.md

```diff
@@ -0,0 +1,133 @@
+# Exercises: Using aux files
+
+We want to explore auxiliary files in these exercises, which allow you to load up contextual information
+alongside your data. 
+
+## 1. Aux files with a plain text file
+
+In this example we will add a simple linear trend to ten years of data, and view the results using
+`ncview`, to confirm that everything works fine. A practical use case here would be to add a varying
+correction to your output data, which is different for each timestep and cannot be expressed purely
+mathematically (you would use custom `script://` step for that case).
+
+First, you'll create a text file with simple integers, one per line, counting up to 120
+
+<details>
+  <summary>Solution</summary>
+  There are many ways to do this. For example:
+
+  ```python
+  numbers = list(range(1, 121))
+  with open("numbers.txt", "w") as f:
+      [f.write(f"{n}\n") for n in numbers]
+  ```
+  Or in pure shell:
+
+  ```bash
+  seq 1 120 > numbers.txt
+  ```
+</details>
+
+Now, we will specify ten years of data in one of our rules. You can use any data you want here:
```

Comment:
"You can use any data you want here": as above, is this really true? Can I use 20 years of data with daily data for this exercise?